src/app.jsx

```diff
@@ -120,6 +120,7 @@ export default class App extends React.Component {
           data={this.state.data}
           selectedAggregationDimension={selectedAggregationDimension}
           colorPack={{
+            columnResizer: 'none',
```

Comment:
@turnerniles Setting default color for the column resizing bars to transparent for now